include/openssl/bn.h

```diff
@@ -452,6 +452,7 @@ OPENSSL_EXPORT int GFp_BN_mod_exp_mont_consttime(BIGNUM *rr, const BIGNUM *a,
 
 /* Private functions */
 
+/* Needs to be kept in sync with `struct BIGNUM` (in `src/bn.rs`) */
```

Comment:
Your "interim" suggestion is exactly what I had in mind as a next step. I'll proceed with that.